ios/Dora/Base.lproj/LaunchScreen.xib

```diff
@@ -1,42 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What happened to this file?

Reply:
It needed to be removed to force it to use my custom launch image instead of the default one.